encord/user_client.py

```diff
@@ -260,12 +260,18 @@ def get_projects(
         return [{"project": OrmProject(p.project), "user_role": ProjectUserRole(p.user_role)} for p in data]
 
     def create_project(
-        self, project_title: str, dataset_hashes: List[str], project_description: str = "", ontology_hash: str = ""
+        self,
+        project_title: str,
+        dataset_hashes: List[str],
+        project_description: str = "",
+        ontology_hash: str = "",
+        source_projects: List = list(),
```

Comment:
Can we type exactly what we expect here? A list of what?

Reply:
done, and added some docs, too